app/javascript/react/container/AirlineShowContainer.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { browserHistory, Link } from 'react-router';
+import AirlineShowTile  from '../components/AirlineShowTile'
+
+class AirlineShowContainer extends Component {
+ constructor(props) {
+   super(props);
+   this.state= {
+     airline: "",
+     destinations: "",
+     rating: ""
+
+   }
+ }
+ componentDidMount(){
+
+ let id = this.props.params.id
+ console.log(id)
+
+ fetch(`/api/v1/airlines/${id}`)
+ .then(response => {
```

Comment:
No change needed here, but just an fyi that you can do this `.then` in one line: 
```
.then(response => response.json())
```
(this syntax automatically returns `response.json()` into the next `.then`, where you have the opportunity to rename it `responseBody` 

😄But again, no changes needed, and this is looking great!